docs/azure_arc_jumpstart/azure_arc_servers/azure_stack_hci/azure_stack_hci_windows/_index.md

```diff
@@ -0,0 +1,226 @@
+    * **ServerClusterEnabled:** Select _$true_ if you have a server cluster created, _$false_ if the not.
+        * If **ServerClusterEnabled** = _$true_, please provide the path to the cluster storage in the following format:
+            * **vmdir:** Disk Letter:\ClusterStorage
+        * If **ServerClusterEnabled** = _$false_, please provide the path to the folder where the VM will be created in the following format:
+            * **vmdir:** Disk Letter:\Folder
+
+    ![Screenshot showing first set of variables in the Powershell script](./03.png)
+
+    As an example:
+    * **DHCPEnabled:** _$false_
+    * **IPAddress:** '192.168.0.11'
+    * **PrefixLenght:** '24'
+    * **DefaultGateway:** '192.168.0.1'
+    * **DNSServer:** '192.168.0.2'
+    * **ServerClusterEnabled:** _$true_
+    * **vmdir:** "C:\ClusterStorage\VMSTORAGE"
+
+2. Environment variables for the VM creation
+
+    * **NodeName:** Provide the name of the node where the VM will be created.
+    * **DomainName:** Provide the name of the domain where the node is added.
+    * **VMName:** Provide the name of the VM.
+    * **VSwitchName:** Provide the name of the Virtual Switch.
+
+    ![Screenshot showing second set of variables in the Powershell script](./04.png)
+
+    As an example:
+    * **NodeName:** 'AZSHCINODE001'
+    * **DomainName:** 'azshci.local'
+    * **VMName:** 'ArcJS-VM1'
+    * **VSwitchName:** 'Internet-vSwitch'
+
+3. Environment variables to onboard the VM to Azure Arc
+
+    * **subID:** Provide the subscriptionID
+    * **appID:** Provide the Service Principal ApplicationID
+    * **secret:** Provide the Service Principal Secret
+    * **tID:** Provide the tenantID
+    * **rgroup:** Provide the Resource Group Name
+    * **location:** Provide the Region
+
+    ![Screenshot showing third set of variables in the Powershell script](./05.png)
+
+    As an example:
+    * **subID:** "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXX"
+    * **appID:** "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXX"
+    * **secret:** "XXXXXXXXXX"
+    * **tID:** "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXX"
+    * **rgroup:** "arc-vms-rg"
+    * **location:** "West Europe"
+
+* After editing the variables, to run the script open PowerShell as an administrator, navigate to the [script folder](https://github.com/microsoft/azure_arc/tree/main/azure_arc_servers_jumpstart/azure_stack_hci/powershell) and run:
+
+  ```powershell
+  .\azstack_hci_vm_deploy.ps1
+  ```
+
+  > **Note: The script takes some minutes to deploy, specially during the .VHDX download**  
+
+    ![Screenshot showing the download of the .VHDX file](./06.png)
+
+  > **Note: You'll be asked to provide the credentials for accessing your host server. If prompted with any other authorization requests (CredSSP, networking related, etc), please answer _Yes_ to all of them.**
+
+    ![Screenshot showing the CredSSP prompt](./07.png)
+    ![Screenshot showing the credentials prompt](./08.png)
+    ![Screenshot showing the network related prompts](./09.png)
```

Comment:
Align the "this can take some time" to the suggested new language and update screenshots